### PR TITLE
REL-3666: Updating NIFS blueprint template rules.

### PR DESCRIPTION
**Pull request by sraaphorst**

### CODE REVIEW

File: bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml [hunk 1]
```diff
@@ -126,7 +129,7 @@ SCIENCE DARKS:
       <container kind="observation" type="Observation" version="2014A-1" subtype="basic" key="dfe11324-c32c-4dac-a70a-06e87d8db3f6" name="NIFS-BP-1">
         <paramset name="Observation" kind="dataObj">
           <param name="title" value="Acq: Before Standard"/>
-          <param name="libraryId" value="0"/>
+          <param name="libraryId" value="1"/>
```
Comment (swalker2m) [inline bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml]: These are referenced in the code, usually.  If the ID changes there should be corresponding changes in the code?

Comment (sraaphorst) [reply bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml]: It appears so, which I'm guessing is why we have the failure. I'm trying to figure out where the changes should be made.

**Comment by sraaphorst** [comment]
This passed, but I'd really like @swalker2m or @tpolecat to review it before I merge it.

File: bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml (hunk 1 above)
Comment (sraaphorst) [reply bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml]: I'm just curious why these IDs would change? What's the significance of migrating 0 to 1, 1 to 2, 2 to 3, 3 to 4, 4 to 5, and 5 to 23? For what reason would we do this?

Comment (cquiroz) [reply bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml]: BTW not that on a few of the skeleton templates there are tests that could break

Comment (sraaphorst) [reply bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml]: @cquiroz Thanks. I have been trying to fix the tests as well, but reached a point on one of thee two (3660) where the algorithm just seems wrong.

**Comment by sraaphorst** [comment]
This is passing all tests. I am going to force-merge it since nobody is available to approve, and I want it in today's test release.

**Pull request merged**